osu.Game/Overlays/Profile/Sections/HistoricalSection.cs

```diff
@@ -1,12 +1,29 @@
 ﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
 
+using osu.Game.Online.API.Requests;
+using osu.Game.Overlays.Profile.Sections.Ranks;
+using osu.Game.Users;
+
 namespace osu.Game.Overlays.Profile.Sections
 {
     public class HistoricalSection : ProfileSection
     {
         public override string Title => "Historical";
 
         public override string Identifier => "historical";
+
+        private readonly ScoreContainer recent;
+
+        public HistoricalSection()
+        {
+            Child = recent = new ScoreContainer.TotalScoreContainer(ScoreType.Recent, "Recent Plays (24h)", "No performance records. :(");
+        }
+
+        public override User User

```

Comment:
I was thinking this with the last PR, but we probably want to use a bindable pattern for `User` here so we don't have to do these increasingly large chains of `set`s. Let me know if you're unsure of how to implement this.